scripts/minikube.sh

```diff
@@ -125,38 +154,40 @@ up)
         install_kubectl
     fi
 
+    disable_storage_addons
```

Comment:
any specific reason for above ?

Reply:
It does not need to deploy any storage components that it deploys by default. Just saving resources by not having them. Not sure if there are potential conflicts, but they did clutter logs about failing to start or something.

Reply:
its better to document it ?

Reply:
Will add some documentation for the function and update its commit message.

Reply:
Thanks!